### DISCUSSION TITLE
install error

### DISCUSSION THREAD

**Discussion by auqf** [harmonycloud/kindling#448] [agent probe]
I follow the link http://www.kindling.space:33215/project-1/doc-35/ to resolve the problem, but failed. though I compiled it by myself
![image](https://user-images.githubusercontent.com/8895540/218631773-fd410fc4-e24b-4763-86fc-8605f76e5ee6.png)

**Comment by dxsup** [comment]
Have you compiled the `kindling-falcolib-probe` successfully? See the output of the script `recompile-module.sh`.
